helm/upp-aggregate-healthcheck/templates/admin-ingress.yaml

```diff
@@ -0,0 +1,24 @@
+{{ if eq .Values.ingress.enabled "true" }}
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: {{.Values.service.name}}-admin-ingress
+  annotations:
+    kubernetes.io/ingress.class: "nginx"
+    ingress.kubernetes.io/ssl-redirect: "true"
+    # type of authentication
+    ingress.kubernetes.io/auth-type: basic
+    # name of the secret that contains the user/password definitions
+    ingress.kubernetes.io/auth-secret: basic-auth
+    # message to display with an appropiate context why the authentication is required
+    ingress.kubernetes.io/auth-realm: "Authentication Required"
+spec:
+  rules:
+    - host: "*.ft.com"
+      http:
+        paths:
+        - path: ~ {{ .Values.env.PATH_PREFIX }}/(?!__).*
```

Comment:
Hmm this isn't perfect as `/__health/` will now require auth, when it shouldn't... Can we do better?

Reply:
`/(?!__).+` maybe?

Reply:
Yes! Your regex fixes the problem